### ISSUE TITLE
Error: Invalid or corrupt jarfile paper.jar

### ISSUE THREAD

**Issue by roaringsundew40** [AlexProgrammerDE/balena-minecraft-server#99]
Hello,

Before I start, I just wanted to say that I'm very new to using Balena Cloud and I'm not entirely sure how everything works (just the basics). So, when you give me an answer please try and give it to me in Layman's terms. Thanks!

This is the first time I am starting up my MC Server and I immediately get this error "Error: Invalid or corrupt jarfile paper.jar".

Here is the full text the comes in the log: 
" 
 mc-server  Setting device hostname to: balenaminecraftserver
 mc-server  
 mc-server  
 mc-server  Starting balenaMinecraftServer...
 mc-server  
 mc-server  Config already copied
 mc-server  Checking server JAR... Found a valid server file. It's called: paper.jar. Use ENABLE_UPDATE to update.
 mc-server  Starting JAR file with: 1G of RAM
 mc-server  Error: Invalid or corrupt jarfile paper.jar
"

I have checked different posts on the internet but they all say to use ENABLE_UPDATE (it also says it in the log). But I don't know how to do that. 

Does anyone know what the error is and how can I fix it? 

Thanks

**Comment by AlexProgrammerDE** [comment]
@roaringsundew40 you should see a tab called "Environment Variables". Create a new variable and set the value to "true"

**Comment by roaringsundew40** [comment]
@AlexProgrammerDE I'm still getting the same Error. Below I have attached a screenshot of what I Did. Is that what I was supposed to do?

![image](https://user-images.githubusercontent.com/58274772/107980999-23a87a00-6f76-11eb-8b58-49f039d40325.png)

**Comment by AlexProgrammerDE** [comment]
@roaringsundew40 yes that looks good. Can you also show me the log again? I think there should have changed something.

**Comment by roaringsundew40** [comment]
@AlexProgrammerDE 
This  is the Log

"
 mc-server  Setting device hostname to: balenaminecraftserver
 mc-server  
 mc-server  
 mc-server  Starting balenaMinecraftServer...
 mc-server  
 mc-server  Forcing server update
 mc-server  Downloading 1.16.5 build 468...
 mc-server  Starting JAR file with: 1G of RAM
 mc-server  Error: Invalid or corrupt jarfile paper.jar
"

**Comment by christian4423** [comment]
I too am getting this error.

**Comment by christian4423** [comment]
Was able to fix. 
In Balena Server log into mc-server and type: 
rm usr/src/serverfiles/paper.jar 
cp paper.jar usr/src/serverfiles

**Comment by roaringsundew40** [comment]
I think it worked but now how do I play on it?

**Comment by AlexProgrammerDE** [comment]
@roaringsundew40 as described in the well written docs on the readme.

**Comment by edu27-eng** [comment]
goeden avond/ ochtend
ik ervaar het zelfde probleem en ik krijg dit niet op die manier opgelost:

> Kon repareren. 
> Log in Balena Server in op mc-server en typ: 
> rm usr / src / serverfiles / paper.jar 
> cp paper.jar usr / src / serverfiles

waar moet ik die regels typen?

**Comment by christian4423** [comment]
Dutch
Hallo, sorry dat ik geen Nederlands spreek. Ik gebruik google translate. [Klik hier](https://imgur.com/iDrwiAI)

English
Hello, sorry I don't speak Dutch. I am using google translate. [Click here] (https://imgur.com/iDrwiAI)

**Comment by hasan0610** [comment]
Ok I got this error many times it has to do with the sad card some sad cards have bad sectors I tried a new sad card same brand same capacity and it worked
